devel/ci/bodhi-ci

```diff
@@ -1199,7 +1199,10 @@ def _build_jobs_list(
                     release=release, depends_on=build_job, buffer_output=buffer_output),
             ])
             jobs.extend(integration_build_jobs)
-        if command in ('all', 'integration'):
+        # we will be skipping integration job due to issues wth alembic
+        # https://github.com/fedora-infra/bodhi/issues/4158
+        # https://github.com/fedora-infra/bodhi/issues/4163
+        if command in ('all', 'integration') and release != 'pip' and release != 'f32':
```

Comment:
integration tests are not blocking the merge, so we could in theory just leave them fail :(

Reply:
No. In my experience, if they are failing and not blocking the merge, these are bad tests. 

Either they are important and should be passing and blocking. Or they are not important and shouldn't be there.

Reply:
Agree, I am fine going that way but we should consider the future of these tests, are we going to invest in them or not.

Reply:
I am leaning in the way of "lets fix pip, keep just pip, rawhide and current stable and agressively(?) upgrade once new version is stable"